pkg/referrerstore/oras/oras.go

```diff
@@ -270,10 +272,18 @@ func (store *orasStore) GetBlobContent(ctx context.Context, subjectReference com
 	// check if blob exists in local ORAS cache
 	isCached, err := store.localCache.Exists(ctx, blobDescriptor)
 	if err != nil {
-		return nil, err
+		logger.GetLogger(ctx, logOpt).Warnf("failed to check if blob [%s] exists in cache: %v", blobDescriptor.Digest.String(), err)
```

Comment:
Curious  have we seen this in a real deployment?

Reply:
Discussed in the PR meeting, we didn't see it yet, but just safe to fix it.